applications/tari_app_grpc/proto/validator_node.proto

```diff
@@ -78,6 +78,13 @@ message InvokeReadMethodRequest{
 
 message InvokeReadMethodResponse {
     optional bytes result = 1;
+    Authority authority = 2;
+}
+
+message Authority {
+    bytes node_public_key =1;
+    bytes signature = 2;
+    optional bytes proxied_by = 3;
```

Comment:
Yeah I also thought that was the case, but for byte arrays, it converts to an Option<Vec<u8>> instead of a Vec<u8> so it's easier to test if it was specified. AFAIK `optional` is still allowed for some types